### PR TITLE
Add adjust parameter to rotate

### PR DESCRIPTION
**Pull request by AlexeyKozhevin**

### CODE REVIEW

**Review by SergeyTsimfer — APPROVED** [review]

**Review by cdtn — APPROVED** [review]

**Review by SergeyTsimfer — COMMENTED** [review]
Comments to notebook:
- Rename the file
- Remove autoreload
- Remove mentions of `rotate_axis` -- there is no such method 
- Swap the `demo_pipeline` and lists of components: first define the pipeline, and then the lists
- You don't need `show_roll`, as far as I see. use `show` instead: before that, copy the `indices` argument to it from `show_roll`
- Also, you can show original image and its augmented versions on the same figure
- Just as a safety thing: please, check that the dtype of all augmented images is `np.float32`

**Review by SergeyTsimfer — CHANGES_REQUESTED** [review]
A gentle ping

File: seismiqb/src/crop_batch.py [hunk 1]
```diff
@@ -939,11 +944,22 @@ def rotate(self, crop, angle, fill_value=0):
...
         fill_value : number
             Value to put at empty positions appeared after crop roration.
         """
         angle = angle if isinstance(angle, (tuple, list)) else (angle, 0, 0)
-        crop = self._rotate(crop, angle[0], fill_value)
+        shape = crop.shape
+        if adjust:
+            if angle[1] != 0 or angle[2] != 0:
+                raise ValueError("Shape adjusting doesn't applicable to 3D rotations")
+            new_shape = adjust_shape_3d(shape=crop.shape, angle=angle)
```
[the remark this answers is not in the document]
Comment (AlexeyKozhevin) [reply seismiqb/src/crop_batch.py:957]: Yes

File: seismiqb/src/crop_batch.py [hunk 2]
```diff
@@ -939,11 +944,22 @@ def rotate(self, crop, angle, fill_value=0):
...
+            crop = cv2.resize(crop, dsize=(new_shape[1], new_shape[0]))
+            if len(crop.shape) == 2:
+                crop = crop[..., np.newaxis]
```
[the remark this answers is not in the document]
Comment (AlexeyKozhevin) [reply seismiqb/src/crop_batch.py:960]: No, `resize` for 3D array of shape `(x, y, 1)` transforms it into `(x', y')` (removes one axis) so I return it

[the remark this answers is not in the document]
Comment (AlexeyKozhevin) [reply seismiqb/src/crop_batch.py:960]: And `crop[..., np.newaxis]` is faster then `crop.reshape(*crop.shape, 1)`

**Pull request merged**